.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 install:
   - pip install --upgrade pip
   - pip install --ignore-installed -r requirements_dev.txt
-  - pip install matplotlib==1.3.1
+  - pip install matplotlib==1.4.1
```

Comment:
Travis moved their infrastructure, and the matplotlib build started failing. There was an issue with it not working with the installed numpy, which had been fixed in newer versions. I don't know how or why it worked before.